projects/collection-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -947,6 +947,21 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
         break;
     }
   }
+
+  onFormStatusChange(form) {
```

Comment:
@vinukumar-vs, We can not use the existing `formStatusChange` method because the behavior of this form is diff from other forms. that's why we have created a separate generic component for this form.